src/lib/Libutil/munge_supp.c

```diff
@@ -63,10 +63,15 @@
 #include <grp.h>
 #endif
 #include "pbs_error.h"
+#include <pthread.h>
 
 
 
 #ifndef WIN32
+static pthread_once_t munge_init_once = PTHREAD_ONCE_INIT;
+#define EBUFSIZE 4096
```

Comment:
or simply use LOG_BUFSIZE - same size i believe

Reply:
LOG_BUF_SIZE is 4096+256. However I think it makes sense to use this value as the buffer size since we cannot possibly log anything beyond this limit.